Typist.swift

```diff
@@ -103,11 +103,12 @@ public class Typist: NSObject {
     
     /// Stops listening to keyboard events. Callback closures won't be cleared, thus calling `start()` again will resume calling previously set event handlers.
     public func stop() {
+        // TODO: this needs a bit more work
```

Comment:
Can you elaborate on "more work" in comment itself? Thanks.

Reply:
I don't know what happened, I did not write it. It was already there when I did my changes.

Reply:
Ah, it seems I've removed it in time you had pulled code and submitted the PR. OK, all clear. I'll merge and release this weekend. Thanks!